src/Common/tests/System/Net/Sockets/TestSettings.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Net.Sockets.Tests
     public static class TestSettings
     {
         // Timeout values in milliseconds.
-        public const int PassingTestTimeout = 10000;
+        public const int PassingTestTimeout = 30000;
```

Comment:
This has been at 10s for months, no?  But it only started causing problems when you added timeouts to some tests in a previous PR?  If so, please leave this as it was and remove those added timeouts.

Reply:
Agreed, I planned on rolling back the timeouts, this was just for my info before I did it.